src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/SentenceCalcType.java

```diff
@@ -64,4 +66,13 @@ public static class PK implements Serializable {
     @Convert(converter = YesNoConverter.class)
     private boolean active;
 
+    public Boolean isAFine() {
```

Comment:
why the nullable result here, especially for an `is` method?  Would prefer
```
"A/FINE".equals(calculationType);
```

Reply:
Done :)